test/Microsoft.AspNetCore.SystemWebAdapters.CoreServices.Tests/RemoteAppClientOptionsTests.cs

```diff
@@ -35,7 +35,11 @@ public void VerifyIsCalled(string apiKeyHeader, string apiKey, string remoteAppU
             {
                 options.ApiKey = apiKey;
                 options.ApiKeyHeader = apiKeyHeader;
-                options.RemoteAppUrl = (remoteAppUrl is null ? null : new Uri(remoteAppUrl, UriKind.Absolute))!;
+
+                if (remoteAppUrl is not null)
```

Comment:
Is this all that's needed to ensure we can handle the new application paths in this PR?

Reply:
no - that just fixes a test failure. The the application paths fix is in the "RemoteModule" changes